src/config/index.ts

```diff
@@ -1,7 +1,6 @@
 export default {
   aad: {
-    clientId: process.env.AAD_CLIENT_ID || '',
-    clientSecret: process.env.AAD_CLIENT_SECRET || '',
+    Secret: process.env.AAD_SECRET || '',
```

Comment:
Would rename the `Secret` local config value name, it's actually storing the value of the name of the secret in secret manager not the actual secret, which is retrieved later on. I'd _potentially_ store it under the AWS section of this file as `aadClientSecretName` just to know that it's referring to an AWS resource?